fast/stages/2-networking-c-nva/data/firewall-rules/heartbeat/default-ingress.yaml

```diff
@@ -0,0 +1,9 @@
+# skip boilerplate check
+
+ingress:
+  landing-ingress-default-deny:
```

Comment:
firewall rules must be globally unique, this rule name is used in three places:
* ./data/firewall-rules/management/default-ingress.yaml
* ./data/firewall-rules/landing/default-ingress.yaml
* ./data/firewall-rules/heartbeat/default-ingress.yaml

Reply:
This was fixed only in heartbeat. Management rule was not fixed.